bibliopixel/drivers/PiWS281X.py

```diff
@@ -42,14 +42,21 @@
     10: 0,  # Technically SPI
 }
 
+STRIP_TYPES = {
+    3: 0x00100800,
+    4: 0x18100800,
+}
+
 
 class PiWS281X(DriverBase):
     """
     Driver for controlling WS281X LEDs via the rpi_ws281x C-extension.
     Only supported on the Raspberry Pi 2 & 3
     """
-    def __init__(self, num, gamma=gamma.NEOPIXEL, c_order=ChannelOrder.RGB, gpio=18,
-                 ledFreqHz=800000, ledDma=5, ledInvert=False, **kwds):
+    def __init__(
+            self, num, gamma=gamma.NEOPIXEL, c_order=ChannelOrder.RGB, gpio=18,
+            ledFreqHz=800000, ledDma=5, ledInvert=False,
+            color_channels=3, brightness=255, **kwds):
```

Comment:
It's sad that we'll still need color_channels here when layout will be getting it eventually as well. But I had to create a custom driver for my 1-bit color display so drivers are always going to be their own thing anyways.